data_processing.tex

```diff
@@ -1,9 +1,11 @@
 \section{Data Processing}
 \label{sec:data_processing}
 
+Data processing is used in this project to help improve feature extraction from sentences as due to their length as much meta data as possible needs to be created to aid classification.
+
   \subsection{Preprocessing}
   \label{sec:preprocessing}
-    \textbf{Renamed from ``Pipelines''}
+    Pipelines are used within the project to help remove extraneous data. They are fully interoperable so can be done any order and allow you to remove stop words, lemmatize or stem the words, encode the sentences or produce windows. All these processes use generators which keeps resource use efficient.
```

Comment:
Good to mention it but do we call them pipelines? Is it a technical term or something we came up with? Sklearn has pipelines which do something similar but we don't use them.